src/tests/multihost/ad/test_id_mapping.py

```diff
@@ -13,24 +17,27 @@ class Testidmaprange(object):
     """
     Test cases for BZ: 1268902
     SSSD doesn't set the ID mapping range automatically
-    @Setup:
-    1. Join to AD using adcli command.
-    2. Add the user using adcli user add.
+    :setup:
+      1. Join to AD using adcli command.
+      2. Add the user using adcli user add.
+    :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1268902
     """
     @pytest.mark.tier1
     def test_001_findrid(self, multihost, get_rid):
         """
-        @Title: IDM-SSSD-TC: Support large AD RIDs automatically: Find RID
-        value from objectSID
+        :title: IDM-SSSD-TC: Support large AD RIDs automatically: Find RID
+         value from objectSID
+        :id: ecb488b2-7694-4e68-9ee7-8833cb7100b1
         """
         (_, rid) = get_rid
         assert rid != 0 or rid is not None
 
     @pytest.mark.tier1
     def test_002_rangelessthansid(self, multihost, get_rid):
         """
-        @Title: IDM-SSSD-TC: Support large AD RIDs automatically: Verify user
-        lookup when ldap idmap range size less than sids
+        :title: IDM-SSSD-TC: Support large AD RIDs automatically: Verify user
```

Comment:
This one is missing :steps: and :expectedresults:

Reply:
Entire test script does not have any  :steps: and :expectedresults:

Reply:
This change was only to change the format of the docstrings that would help to auto import test cases and results into polarion. Where steps and expectedresults were available, I have formated them.